app/main.py

```
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd, source, target = command.split()
+
+        if cmd != "cp" or source == target:
+            return
+
+        with open(source, "r") as src, open(target, "w") as dst:
+            dst.write(src.read())
+
+    except Exception:
```

Catching a generic `Exception` is generally discouraged because it can suppress unexpected errors (like a typo in a variable name) and make your code harder to debug. It is better practice to catch only the specific exceptions you anticipate, such as `ValueError` for an incorrect command format, or `IOError`/`FileNotFoundError` for file-related issues.